avocado/core/job.py

```diff
@@ -120,8 +120,9 @@ def __init__(self, args=None):
         #: has not been attempted.  If set to an empty list, it means that no
         #: test was found during resolution.
         self.test_suite = None
-        self.job_pre_post_dispatcher = dispatcher.JobPrePostDispatcher()
-        output.log_plugin_failures(self.job_pre_post_dispatcher.load_failures)
+
+        #: A job may not have a dispatcher for pre/post tests execution plugins
+        self.job_pre_post_dispatcher = None
```

Comment:
I think we should start putting those internal values with `_` prefix and this one looks like a good candidate.